### PR TITLE
post: ng 9.1

### PR DESCRIPTION
**Pull request by cexbrayat**
TODO:

- [x] release link
- [x] post date

### CODE REVIEW

File: _posts/2020-03-25-what-is-new-angular-9.1.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+### Better Jest support
+
+Until now, zone.js automatically added support for Jasmine and Mocha
+when importing `zone-testing` (the CLI imports it in the `test.ts` file of your project).
+It now also offers built-in support for [Jest](https://jestjs.io/),
+a popular testing framework.
+
+This support already existed via the community library
+[`jest-preset-angular`](https://github.com/thymikee/jest-preset-angular),
+which is still needed, but can now avoid to mingle with Zone.js itself
```
Comment (clacote) [inline _posts/2020-03-25-what-is-new-angular-9.1.md]: avoid "you" to mingle with, no?

File: _posts/2020-03-25-what-is-new-angular-9.1.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+and only focus on the Jest part.
+
+### MessagePort
+
+Zone.js can now handle the
+[Channel Message API](https://developer.mozilla.org/en-US/docs/Web/API/Channel_Messaging_API).
+If you listen to messages in you Angular application,
```
Comment (clacote) [inline _posts/2020-03-25-what-is-new-angular-9.1.md]: you -> your

File: _posts/2020-03-25-what-is-new-angular-9.1.md (hunk 1 above)
Comment (cexbrayat) [reply _posts/2020-03-25-what-is-new-angular-9.1.md]: I meant the the thrid party library: I'll clarify

File: _posts/2020-03-25-what-is-new-angular-9.1.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+### Better Jest support
+
+Until now, zone.js automatically added support for Jasmine and Mocha
+when importing `zone-testing` (the CLI imports it in the `test.ts` file of your project).
+It now also offers built-in support for [Jest](https://jestjs.io/),
+a popular testing framework.
+
+This support already existed via the community library
+[`jest-preset-angular`](https://github.com/thymikee/jest-preset-angular),
+which is still needed, but the library now avoid to mingle with Zone.js itself
```
Comment (jnizet) [inline _posts/2020-03-25-what-is-new-angular-9.1.md]: avoid -> avoids

File: _posts/2020-03-25-what-is-new-angular-9.1.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+and only focus on the Jest part.
```
Comment (jnizet) [inline _posts/2020-03-25-what-is-new-angular-9.1.md]: focuses

File: _posts/2020-03-25-what-is-new-angular-9.1.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+are resolved when calling `tick`.
+You can now give the extra option to `tick` to force it to resolve
+only the currently installed timeouts.
+
+You can also check out
+[our blog post about the CLI v9.1](TODO)
+to see what's new there.
+
+I'm not sure I can publicly talk about it yet,
+but some long-awaited work should start soon,
```
Comment (jnizet) [inline _posts/2020-03-25-what-is-new-angular-9.1.md]: I'd rephrase this as 
> I'm not sure I can publicly talk about it yet,
> so all I will say is that long-awaited work should start soon,
